docker-compose-configuration/rds-local-apps/rds-local/plugin.properties

```diff
@@ -0,0 +1,4 @@
+plugin.id=rds-local
+plugin.provider=Metaphacts
+plugin.version=0.0.1
```

Comment:
please make this `1.0.0` for a proper initial release :-)

Reply:
No difference but it's done.